lib/types/operation.js

```diff
@@ -599,7 +600,7 @@ Operation.prototype.getBody = function (headers, args, opts) {
             bodyParam.append(key, value.value);
           } else {
             if (Array.isArray(value)) {
-              bodyParam.append(key, this.encodeQueryCollection(param.collectionFormat, key, value));
+              bodyParam.append(key, this.encodeQueryCollection(param.collectionFormat, key, value).split('=').slice(1).join('='));
```

Comment:
I'm not sure why we would split like this.  Seems like this would be undoing something that we did already?